docs/configs/site.js

```diff
@@ -45,11 +45,16 @@ export default {
         children: [
           {
             key: 'docs0',
-            text: 'latest(3.0.1)',
+            text: 'latest(3.1.0)',
             link: '/en-us/docs/latest/user_doc/about/introduction.html',
           },
           {
             key: 'docs1',
+            text: '3.0.1',
+            link: '/en-us/docs/3.0.1/user_doc/guide/quick-start.html',
+          },
+          {
+            key: 'docs2',
```

Comment:
add new key `docs2`

Reply:
Looking great, we should keep both 3.0.x and 3.1.x